plugins/processors/round/sample.conf

```
@@ -0,0 +1,11 @@
+# Round numerical fields
+[[processors.round]]
+  ## Precision to round to.
+  ## A positive number indicates rounding to the right of the decimal separator (i.e. the fractional part).
+  ## A negative number indicates rounding to the left of the decimal separator.
+  # precision = 0
+
+  ## Round only numeric fields matching the filter criteria below.
+  ## Excludes takes precedence over includes.
+  # fields_include = []
```

Maybe this makes the default behavior more clear?

```suggestion
  # fields_include = ["*"]
```